src/app/features/fss-search/fss-search.component.ts

```diff
@@ -204,9 +204,12 @@ export class FssSearchComponent implements OnInit {
       errmsg = err.error.message;
     }
     if(this.activeSearchType == this.SearchTypeEnum.SimplifiedSearch)
-      {
-        this.showMessage("error", "There has been an error", "please contact customer services");
-      }
+      if(err.error.statusCode == this.TooManyRequest){
```

Comment:
Does the changed error message apply to only simplified search? It should probably also apply to advanced search if not already.

Could you please also tidy up the `if` statement/blocks

Reply:
In simplified search common message is shown to contact administrator and that is why we explicitly checked for 429.  
whereas In advanced search whatever response we get from API that is shown.

Reply:
Yes please handle it the same way for Advanced Search too